helm/values.yaml

```diff
@@ -2018,12 +2018,15 @@ ui:
   #    secret:
   #      secretName: gravitee-config-secret-name
 
+# NOTE: Kafka Console is in tech preview and requires a private image.
+# Contact your technical account manager for access and the correct image reference.
 kafkaConsole:
   enabled: false
   name: kafkaConsole
   replicaCount: 1
   image:
-    repository: graviteeio/apim-kafka-console
+    repository: graviteeio/apim-kafka-console # Example - update with your private registry
+    # tag: latest # Example - update with your version
```

Comment:
Not sure if it's necessary to put an comment `# Example` here, no? we don't do it for others. The values.yaml here are an example file in themselves imho